config/routes.rb

```diff
@@ -549,8 +549,6 @@ def scoped_navigation_routes(context, navigation)
         namespace :questions do
           get "show_xml", to: "confirmation#show_xml"
           get "explain_calculations", to: "confirmation#explain_calculations"
-          get "pending_federal_return", to: "pending_federal_return#edit"
-          get "canceled_data_transfer", to: "canceled_data_transfer#edit"
```

Comment:
Why are these removed?

Reply:
A: because this section is not state specific, the addition below is to "replace" these lines